create-and-save-map/src/main/java/com/esri/arcgismaps/sample/createandsavemap/screens/MainScreen.kt

```diff
@@ -0,0 +1,462 @@
+                // show the "Edit map" button only when the bottom sheet is not visible
+                if (!controlsBottomSheetState.isVisible) {
+                    ExtendedFloatingActionButton(
+                        modifier = Modifier
+                            .align(Alignment.BottomEnd)
+                            .padding(bottom = 36.dp, end = 24.dp),
+                        onClick = {
+                            composableScope.launch {
+                                controlsBottomSheetState.show()
+                            }
+                        })
+                    {
+                        Icon(Icons.Filled.Edit, contentDescription = "Edit map button")
+                        Spacer(Modifier.padding(8.dp))
+                        Text("Edit Map")
+                    }
+                }
+
+                if (controlsBottomSheetState.isVisible) {
+                    ModalBottomSheet(
+                        modifier = Modifier.wrapContentHeight(),
+                        sheetState = controlsBottomSheetState,
+                        onDismissRequest = {
+                            composableScope.launch {
+                                controlsBottomSheetState.hide()
+                            }
+                        }
+                    ) {
+                        Column(
+                            Modifier
+                                .padding(12.dp)
+                                .navigationBarsPadding()
+                        ) {
+                            BasemapDropdown(
+                                mapViewModel.basemapStyle,
+                                mapViewModel.stylesMap,
+                                mapViewModel::updateBasemapStyle
+                            )
+                            Spacer(Modifier.size(8.dp))
+
+                            LayersDropdown(
+                                mapViewModel.availableLayers,
+                                mapViewModel.arcGISMap,
+                                mapViewModel::updateActiveLayers
+                            )
+
+                            HorizontalDivider(Modifier.padding(vertical = 12.dp, horizontal = 8.dp))
+
+                            Row(
+                                Modifier.fillMaxWidth(),
+                                verticalAlignment = Alignment.CenterVertically,
+                                horizontalArrangement = Arrangement.SpaceBetween
+                            ) {
+                                Text(text = "Name: ", Modifier.padding(8.dp))
+                                TextField(
+                                    value = mapViewModel.mapName,
+                                    onValueChange = { newName -> mapViewModel.updateName(newName) },
+                                    Modifier.width(270.dp),
+                                    singleLine = true
+                                )
+                            }
```

Comment:
![Screenshot 2024-09-26 at 4 24 53 PM](https://github.com/user-attachments/assets/97ae7718-4164-4c03-9478-5e26fb60ce45)


For UI which performs form style selection (name, title, selection property, list item), I like that you are using the `TextField` to act as placeholders for the input via dropdowns/keyboard input. 

As an enhancement, I would suggest using the `label` tag instead of using a Row with the label text. The `supportingText` could also be helpful to provide additional descriptions for the input property. This is more subjective, but I find `OutlineTextField` more appealing than `TextField`. 

Also, word of caution, always try to avoid using hard-coded width/height size values as they may work and fit well on your screen/device, but may not necessarily work on different devices. A fixed width would not perform well, especially on device rotation. In this case, I used fillMaxWidth: 

```suggestion
                            OutlinedTextField(
                                modifier = Modifier.fillMaxWidth(),
                                value = mapViewModel.mapName,
                                label = { Text(text = "Enter map name: ")},
                                onValueChange = { newName -> mapViewModel.updateName(newName) },
                                singleLine = true,
                                supportingText = { }
                            )
```

Reply:
If possible could you update this pattern to all the TextField inputs?